src/app/components/Form/AmountInput/index.tsx

```diff
@@ -59,7 +59,7 @@ const amounts = [10, 25, 50, 75, 100];
 interface AmountSelectorProps {
   asset?: Asset;
   maxAmount?: string;
-  onChange: (value: string) => void;
+  onChange: (value: string, isTotal: boolean) => void;
```

Comment:
Why `isTotal` is not optional here? I know you set `isTotal` later in `handleChange` but the function signature is exactly the same as in `Props`.